From 41e17f54b178f9fc23a5f06583e572a19f933442 Mon Sep 17 00:00:00 2001
From: Dong Aisheng <aisheng.dong@nxp.com>
Date: Thu, 11 May 2017 16:46:54 +0800
Subject: [PATCH 1773/5242] MLK-14823 dts: imx6sx-sabreauto: correct cd and wp
 pin

commit  e16d2b56c29acb71f77215e74524053a494f1dbf from
https://source.codeaurora.org/external/imx/linux-imx.git

Correct CD/WP pin in dts. Otherwise the card detection and write
protection function does not work.

Signed-off-by: Dong Aisheng <aisheng.dong@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6sx-sabreauto.dts |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/imx6sx-sabreauto.dts b/arch/arm/boot/dts/imx6sx-sabreauto.dts
index 9d7e20a..70c255b 100644
--- a/arch/arm/boot/dts/imx6sx-sabreauto.dts
+++ b/arch/arm/boot/dts/imx6sx-sabreauto.dts
@@ -951,8 +951,8 @@
 			MX6SX_PAD_SD3_DATA5__USDHC3_DATA5	0x17059
 			MX6SX_PAD_SD3_DATA6__USDHC3_DATA6	0x17059
 			MX6SX_PAD_SD3_DATA7__USDHC3_DATA7	0x17059
-			MX6SX_PAD_KEY_COL0__GPIO2_IO_10		0x17059 /* CD */
-			MX6SX_PAD_KEY_ROW0__GPIO2_IO_15		0x17059 /* WP */
+			MX6SX_PAD_USB_H_DATA__GPIO7_IO_10       0x17059 /* CD */
+			MX6SX_PAD_LCD1_DATA18__GPIO3_IO_19      0x17059 /* WP */
 		>;
 	};
 
-- 
1.7.9.5

